openapi/sms/v1-review/docs/readme.md

```diff
@@ -0,0 +1,262 @@
+---
+stoplight-id: cz3ydei53qosx
+---
+
+# Emarsys SMS Partner API Documentation
+
+SAP Emarsys empowers marketers around the world to create a truly personalized omnichannel experiences that delivers business outcomes. The Emarsys SMS Partner API allows partners to integrate onto the Emarsys SMS channel. The Partner API uses a RESTful approach.
+
+## API Specification
+
+The API specification has been created using OpenAPI 3.0.3. It is available to be viewed and exported using the links below:
+
+- [Partner Service Specification](../partner-service.yaml)
+- [Partner Callbacks Specification](../partner-callbacks.yaml)
+
+The current functionality supported is:
+
+- [Self-service on-boarding for the client end user](#client-onboarding)
+- [Sending SMS messages triggered by the Emarsys platform](#outbound-messages)
+- [Recording delivery reports of sent messages](#delivery-reports)
+- [Receiving inbound SMS messages](#inbound-messages)
+
+## Glossary
```

Comment:
Could we extend the glossary with a short definition of what inboud/outbound message is? 

Maybe is trivial, but for me it would be useful